ports/stm32/machine_uart.c

```diff
@@ -147,7 +147,15 @@ STATIC const mp_irq_methods_t pyb_uart_irq_methods = {
 STATIC void pyb_uart_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind) {
     pyb_uart_obj_t *self = MP_OBJ_TO_PTR(self_in);
     if (!self->is_enabled) {
-        mp_printf(print, "UART(%u)", self->uart_id);
+        #ifdef LPUART1
+        if (self->uart_id == PYB_LPUART_1) {
+            mp_printf(print, "UART('MP1')");
```

Comment:
Shouldn't this be LP1?

Reply:
I think it should be 'LP1", also line 182. I will review, rebase and retest this week